config/packages/scheb_2fa.yaml

```diff
@@ -12,7 +12,7 @@ scheb_two_factor:
         enabled: true
         server_name: '%env(APP_HOSTNAME)%'
         issuer: Packagist
-        window: 1
+        leeway: 10
```

Comment:
wasn't able to test that locally yet ...

https://github.com/scheb/2fa/commit/e9f23f5b438ad2d48d99164fc98ae14a8d538fbd

Reply:
Works 👍🏻